config/providers/safespring/capi/common-config.yaml

```
@@ -0,0 +1,9 @@
+networkPolicies:
+  global:
+    externalLoadBalancer: false
```

```suggestion
    externalLoadBalancer: false
    ingressUsingHostNetwork: true
  ingressNginx:
    ingressOverride:
      enabled: true
      ips:
        - 0.0.0.0/0
```
Is also needed to make the netpols work.

---

I never had to do this. Could you explain why this is necessary? It looks like disabling host ports is adding a lot of extra baggage? ELI5 the benefit. 😄

---

Great write-up and preserving the client traffic is absolutely what we should aim for. Thanks for the clarification.

The baggage I was referring to was only config related since we needed more overrides.

It should be stated that this is going to be untested from the release QA point of view. But if you have tested this setup on Safespring with kube-vip I'm fine.

Note that `ingressUsingHostNetwork: true` was already the case since this was used in Safespring Kubespray.

https://github.com/elastisys/compliantkubernetes-apps/pull/2890/commits/2064bfebd620cfb3cbfe94ba71d4ad7384f30824